server/server/models/events.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Column, DateTime, String, Integer, func, ForeignKey
+from sqlalchemy.orm import relationship
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class Event(Base):
+    __tablename__ = 'events'
+    id = Column(Integer, primary_key=True)
+    name = Column(String, nullable=False, unique=True)
+    lng = Column(String, nullable=False)
+    lnt = Column(String, nullable=False)
+    description = Column(String, nullable=False)
+    data_start = Column(String, nullable=False)
+    data_end = Column(String, nullable=False)
```

Comment:
`date`, not `data`. And it's better to reverse words order: e.g. `start_date`. Also use correct data type.